blocks/resources/block.php

```diff
@@ -106,23 +112,45 @@ function render_callback( array $attributes ) : string {
 			esc_attr( $anchor ),
 			esc_html( $need_name )
 		);
-		$html  .= '<div class="resources__item-wrapper">';
+		$html  .= '<ul class="resources__item-wrapper">';
+
 		foreach ( $need['Resources'] as $resource ) {
+			$html .= '<li class="resources__item">';
+
 			$html .= sprintf(
-				'<div class="resources__item">
-					<span class="resources__tag resources__tag--%5$s">%4$s</span>
-					<h3 class="resources__item-title">%1$s</h3>
-					%2$s
-					<p><a href="%3$s" class="resources__item-link">%3$s</a></p>
-				</div>',
-				esc_html( $resource['Resource Title'] ),
-				wp_kses_post( markdown_to_html( $resource['Resource Details'], $markdown_parser ) ),
+				'<h3 class="resources__item-title"><a href="%s">%s</a></h3>',
 				esc_url( $resource['Link to Resource'] ?? '' ),
+				esc_html( $resource['Resource Title'] ),
+			);
+
+			if ( count( $resource['Group'] ) ) {
+				$group_names = [];
+
+				foreach ( $resource['Group'] as $group ) {
+					if ( isset( $group['Website'] ) ) {
+						array_push( $group_names, sprintf( '<a href="%s">%s</a>', esc_url( $group['Website'] ), esc_html( $group['Group Name'] ) ) );
+					} else {
+						array_push( $group_names, $group['Group Name'] );
+					}
+				}
+
+				$html .= sprintf(
+					'<p class="resources__item-group">Provided by %s</p>',
```

Comment:
For strings like this, it's best WP practice to localize them:
```suggestion
					'<p class="resources__item-group">' . esc_html__( 'Provided by %s' ) . '</p>',
```